src/hooks/listHooks.js

```diff
@@ -35,23 +35,25 @@ const getListType = (query, listTypesSettings) => {
 };
 
 const EventSectionHeader = ({ item, navigation, options, query }) => (
-  <SectionHeader
-    title={momentFormat(item, 'DD.MM.YYYY dddd')}
-    onPress={
-      navigation
-        ? () =>
-            navigation.push(ScreenName.Index, {
-              title: texts.homeTitles.events,
-              query,
-              queryVariables: {
-                ...options.queryVariables,
-                dateRange: [momentFormat(item, 'YYYY-MM-DD'), momentFormat(item, 'YYYY-MM-DD')]
-              },
-              rootRouteName: ROOT_ROUTE_NAMES.EVENT_RECORDS
-            })
-        : undefined
-    }
-  />
+  <View style={styles.sectionHeader}>
+    <SectionHeader
```

Comment:
couldn't this be the `containerStyle` of the `SectionHeader` instead of a new `View` wrapping here?

Reply:
I updated it to `sectionHeaderContainer`, I think this name is better.

Reply:
setting the `containerStyle` with `paddingTop` should overwrite the default values of the styled component.